types/ui/tabs.d.ts

```diff
@@ -1,150 +1,95 @@
  * tabs.bind(html);
  * ```
  */
-declare class TabsV2 {
+declare class Tabs {
   /**
-   * The value of the active tab
+   * @param navSelector     - The CSS selector used to target the navigation element for these tabs
+   * @param contentSelector - The CSS selector used to target the content container for these tabs
+   * @param initial         - The tab name of the initially active tab
+   * @param callback        - An optional callback function that executes when the active tab is changed
    */
-  active: string;
+  constructor({ navSelector, contentSelector, initial, callback }: Tabs.Options);
 
   /**
-   * A callback function to trigger when the tab is changed
+   * The value of the active tab
    */
-  callback: Function;
+  active: string | undefined;
 
   /**
-   * A reference to the HTML container element of the tab content
+   * A callback function to trigger when the tab is changed
    */
-  protected _content: HTMLElement;
+  callback: ((event: null, tabs: Tabs, tabName: string) => unknown) | null | undefined;
 
   /**
-   * The CSS selector used to target the tab content element
+   * The CSS selector used to target the tab navigation element
    */
-  protected _contentSelector: string;
+  protected _navSelector: string;
 
   /**
    * A reference to the HTML navigation element the tab controller is bound to
+   * @defaultValue `null`
    */
-  protected _nav: HTMLElement;
+  protected _nav: HTMLElement | null;
 
   /**
-   * The CSS selector used to target the tab navigation element
+   * The CSS selector used to target the tab content element
    */
-  protected _navSelector: string;
-
-  constructor(options: TabV2Options);
+  protected _contentSelector: string | undefined;
 
   /**
-   * Activate a new tab by name
-   * @param tabName -
-   * @param triggerCallback -
+   * A reference to the HTML container element of the tab content
+   * @defaultValue `null`
    */
-  activate(tabName: string, options?: { triggerCallback?: boolean }): void;
+  protected _content: HTMLElement | null;
 
   /**
    * Bind the Tabs controller to an HTML application
-   * @param html -
    */
   bind(html: HTMLElement): void;
 
+  /**
+   * Activate a new tab by name
+   * @param tabName         - Name of the tab to activate
+   * @param triggerCallback - Whether or not to trigger the callback
+   *                          (default: `false`)
+   */
+  activate(tabName: string, { triggerCallback }?: { triggerCallback?: boolean }): void;
+
   /**
    * Handle click events on the tab navigation entries
    * @param event - A left click event
    */
   protected _onClickNav(event: MouseEvent): void;
 }
+
+declare namespace Tabs {
+  interface Options {
+    navSelector: string;
+    contentSelector?: string;
+    initial?: string;
+    callback?: ((event: null, tabs: Tabs, tabName: string) => unknown) | null;
+  }
+}
+
+declare const TabsV2: typeof Tabs;
```

Comment:
Is this referenced only in our own types or actually defined in `foundry.js`?

Reply:
it's defined exactly like this in `foundry.js`:
```js
const TabsV2 = Tabs;
```